todoapp/utils/multer.js

```diff
@@ -0,0 +1,27 @@
+const multer = require("multer");
```

Comment:
you can try and rename the file name from multer to something more generic just in case you are not using multer to upload. Like imageUpload.js